charts/kubeovn-operator/templates/deployment.yaml

```diff
@@ -7,17 +7,18 @@ metadata:
   name: kubeovn-operator-controller-manager
   namespace: {{ .Release.Namespace }}
 spec:
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      {{- include "kubeovn-operator.labels" . | nindent 6 }}
+      kubectl.kubernetes.io/default-container: manager
       control-plane: controller-manager
+      app.kubernetes.io/instance: kubeovn-operator
```

Comment:
Maybe we can use `selectorLabels` in `_helper.tpl`?  I found there are some defined variables like other addon have.

Reply:
good point, i can switch to that